### PR TITLE
Add README.md

### PR DESCRIPTION
**Pull request by n-valchev**
Add a README.md file containing a brief documentation about FlyCI Wingman Action

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+```yaml
+- uses: fly-ci/wingman-action@main
+  if: failure()
+```
+
+### Requirements
+
+#### FlyCI Application
+
+You have to have [FlyCI app](https://github.com/apps/flyci-prod/installations/select_target) installed for the repo in order for the action to work.
```
Comment (harveysmurf) [inline README.md]: Can you ask chatGPT for alternatives so we avoid "you have to have"

**Review by harveysmurf — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (n-valchev) [reply README.md]: how about now

File: README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+## What's new
+
+Please refer to the [release page](https://github.com/fly-ci/wingman-action/releases/latest) for the latest release notes.
+
+## Usage
+
+Add `fly-ci/wingman-action` to your workflow job as the **last step**:
+
+```yaml
+- uses: fly-ci/wingman-action@main
```
Comment (ligaz) [inline README.md:26]: Is it recommended to refer the version from `main`?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+```diff
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  ci:
+    runs-on: flyci-macos-14-m2
```
Comment (ligaz) [inline README.md]: Let's use `ubuntu-latest` here as a more prime example.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+
+      - name: Install dependencies
+        run: npm install
```
Comment (ligaz) [inline README.md]:
```suggestion
        run: npm ci
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+Currently, FlyCI Wingman is in Beta and **free of charge**. For more information, please see our [Billing](https://flyci.net/docs/about/billing#billing-process) page.
+
+## License
+
+The scripts and documentation in this project are released under the [MIT License](LICENSE)
+
+## Feedback Channels
+
+If you would like to report a potential issue, get help, or recommend a feature, please join our [Discord Community](https://github.com/fly-ci/wingman-action/discussions).
```
Comment (radeva) [inline README.md]: Discord link doesn't seem correct. The correct invite link is: https://discord.gg/JyCjh439da

File: README.md (hunk 2 above)
Comment (n-valchev) [reply README.md:26]: My plan is to update it once we have a (plan to) release. As of now this is the only branch we maintain, so if one comes here and wants to try it out (even we haven't released it) they would be able to using this documentation.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+```yaml
+- uses: fly-ci/wingman-action@main
+  if: failure()
+```
+
+### Requirements
+
+#### FlyCI Application
+
+It's required to have [FlyCI app](https://github.com/apps/flyci-prod/installations/select_target) installed for the repo in order for the action to work.
```
Comment (radeva) [inline README.md]:
```suggestion
It's required to have the [FlyCI app](https://github.com/apps/flyci-prod/installations/select_target) installed for the corresponding repository for the action to work properly.
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+#### Assign required permissions
+
+In order to get authenticated in front of the FlyCI API, the action has to obtain a short-lived access token using a GitHub OpenID Connect (OIDC) token.
+For this to work, you have to grant `id-token: write` permission to the job:
```
Comment (radeva) [inline README.md]:
```suggestion
To do so, it is required to grant `id-token: write` permission to the job:
```

**Review by radeva — DISMISSED** [review]

**Review by radeva — APPROVED** [review]

**Pull request merged**